src/components/AssetManagementPanel/AssetManagementForm/FormVariants/AcceptSurrenderForm/AcceptSurrenderForm.tsx

```diff
@@ -11,8 +11,11 @@ import {
   showDocumentTransferMessage,
```

Comment:
Just some minor comments on the nomenclatures... 

The state that the token is in when it is with the token registry contract should be called "surrendered"
The state that the token is at 0x0 should be called "destroyed"
The state that the token is returned to a user (regardless if its a title escrow) can be called "returned"

Assuming one agrees with that, this form should be called, `handleSurrenderForm` since it can progress in two direction:
1. surrendered -> destroyed (currently called `acceptSurrendering` which is incorrect because it has been surrendered)
2. surrendered -> returned (currently called `rejectSurrendering` which is incorrect because the document is already in the possession of the token registry)